contracts/router/src/contract.rs

```diff
@@ -20,10 +23,7 @@ pub fn migrate(
     _env: Env,
     _msg: Empty,
 ) -> Result<Response, axelar_wasm_std::ContractError> {
-    // any version checks should be done before here
-
-    cw2::set_contract_version(deps.storage, CONTRACT_NAME, CONTRACT_VERSION)?;
-
```

Comment:
Are we setting the version somewhere else now?

Reply:
hmm, that line got lost. Added it back in